### PR TITLE
Manticore 0.1.6

### PR DESCRIPTION
**Pull request by mossberg**
- Updates changelog
- updates readme
- Bumps version numbers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,42 @@
...
+- [chowdaryd](https://github.com/chowdaryd)
+
+### Deprecated
+
+### Added
+
+- Ethereum support in the command line (Solidity files)
+- --version flag to command line
+- x64/64: Improved support for PCMPXSTRX instruction family
+- [alpha] Ethereum EVM dis/assembly APIs
```
Comment (mossberg) [inline CHANGELOG.md]: i wanted to express here that this is an initial version of the feature, it is not stable or production ready, but we want feedback

Comment (dguido) [reply CHANGELOG.md]: _Everything_ in Ethereum is an initial version, is not stable or production ready, and looking for feedback. I say go without mentioning it. ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,42 @@
...
+- [alpha] Ethereum EVM analysis APIs
+
+### Changed
+
+- Workspace .txt file extension changed to .input
+
+### Fixed
+
+- Unserializing Linux states with special files opened
```
Comment (yan) [inline CHANGELOG.md]: "Deserializing"?

Comment (mossberg) [reply CHANGELOG.md]: 🤔 https://github.com/pyca/cryptography/issues/3338
yup, agreed

**Pull request merged**